apps/next/src/components/ui/card/food-card.tsx

```diff
@@ -32,31 +36,58 @@ const FoodCardContent = React.forwardRef<
 >(({ dish, ...divProps }, ref) => {
   const IconComponent = getFoodIcon(dish.name) ?? Utensils;
 
-  const caloricInformationAvailable: boolean = dish.nutritionInfo.calories != null
-    && dish.nutritionInfo.calories.length > 0;
+  const utils = trpc.useUtils();
+  const logMealMutation = trpc.nutrition.logMeal.useMutation({
+    onSuccess: () => {
+      alert(`Added ${formatFoodName(dish.name)} to your log`);
+      utils.nutrition.invalidate();
+    },
+    onError: (error: Error) => {
+      alert(error.message);
```

Comment:
Prefer `console.error`ing this.

```suggestion
      console.error(error.message);
```